### DISCUSSION TITLE
Navigation of google maps on ios

### DISCUSSION THREAD

**Discussion by akiratoshi** [zepp-health/zeppos-forum#20] [Q&A] [answered]
Hi,
I see in the Zepp app allows receiving notifications from Google maps. Is it possible to get navigational notifications from the Zepp app and then pass them on to the app on the device? I think we will need to convert data(since the information I know the data passed from the Zepp app is binary data). But, if we have a way to get the information, it will be a lot easier.  
I see for Android now it has that, the question is why iOS doesn't have it? Having technical difficulties?

Thank you everyone for your interest and time

**Comment by MyoungXUE** [comment, marked as answer]
Thank you for your support of the Developer Forum!

Sorry for the long wait, because I have been looking for the relevant colleagues to confirm the problem.

This is a very good idea, but currently technically impossible to implement.

The firmware has not yet opened the notification-related interface for the Device App, which is the most critical reason.

**Reply by kartikjagdale** [reply]
I am willing to contribute to this app development, let me know if anything needed.

**Reply by MyoungXUE** [reply]
> I am willing to contribute to this app development, let me know if anything needed.

Thank you for your enthusiasm!

At this stage Zepp OS's open capabilities do not support blocking notifications from the system, we will consider planning for a later version.
